rkt/image/filefetcher.go

```diff
@@ -85,13 +89,21 @@ func (f *fileFetcher) getVerifiedFile(aciPath string, a *asc) (*os.File, error)
 	if err != nil {
 		return nil, errwrap.Wrap(errors.New("error opening signature file"), err)
 	}
-	defer func() { maybeClose(ascFile) }()
+	defer func() {
+		if ascFile != nil {
+			ascFile.Close()
+		}
+	}()
 
 	aciFile, err := os.Open(aciPath)
 	if err != nil {
 		return nil, errwrap.Wrap(errors.New("error opening ACI file"), err)
 	}
-	defer func() { maybeClose(aciFile) }()
+	defer func() {
+		if aciFile != nil {
+			aciFile.Close()
+		}
+	}()
```

Comment:
we are returning `aciFile` of type `*os.File`, hence it is the caller's responsibility to close it.